tests/inverted-index-test.js

```diff
@@ -8,104 +8,107 @@ import allMalformedBooks from '../fixtures/allMalformedBooks.json';
+      .toEqual([false, 'Malformed!']);
+      expect(InvertedIndex.validateBookObjs(oneMalformedBook))
+      .toEqual([false, 'Malformed!']);
     });
 
     it('returns [true] if all books in file is valid', () => {
-      expect(InvertedIndex.checkBooks(validFile)).toEqual([true]);
-      expect(InvertedIndex.checkBooks(anotherValidFile)).toEqual([true]);
+      expect(InvertedIndex.validateBookObjs(validFile)).toEqual([true]);
+      expect(InvertedIndex.validateBookObjs(anotherValidFile)).toEqual([true]);
     });
   });
 
   describe('InvertedIndex.validateFileContent(fileContent)', () => {
     it("returns [false, 'empty'] if fileContent is empty", () => {
       expect(InvertedIndex.validateFileContent(emptyFile))
-      .toEqual([false, 'empty']);
+      .toEqual([false, 'Empty!']);
     });
 
     it(`returns [false, 'invalid']
      if fileContent is not valid JSON array`, () => {
       expect(InvertedIndex.validateFileContent(emptyObject))
-      .toEqual([false, 'invalid']);
-      expect(InvertedIndex.validateFileContent(validFile)).toEqual([true]);
+      .toEqual([false, 'Invalid!']);
     });
 
-    it(`calls InvertedIndex.checkBooks
+    it(`calls InvertedIndex.validateBookObjs
      if fileContent is non-empty valid JSON array`, () => {
-      spyOn(InvertedIndex, 'checkBooks');
+      spyOn(InvertedIndex, 'validateBookObjs');
       InvertedIndex.validateFileContent(validFile);
       InvertedIndex.validateFileContent(allMalformedBooks);
-      expect(InvertedIndex.checkBooks).toHaveBeenCalled();
-      expect(InvertedIndex.checkBooks).toHaveBeenCalledWith(validFile);
-      expect(InvertedIndex.checkBooks).toHaveBeenCalledWith(allMalformedBooks);
+      expect(InvertedIndex.validateBookObjs)
+      .toHaveBeenCalled();
+      expect(InvertedIndex.validateBookObjs)
+      .toHaveBeenCalledWith(validFile);
+      expect(InvertedIndex.validateBookObjs)
+      .toHaveBeenCalledWith(allMalformedBooks);
     });
 
-    it(`returns InvertedIndex.checkBooks(fileContent)
+    it(`returns InvertedIndex.validateBookObjs(fileContent)
      if fileContent is non-empty valid JSON array`, () => {
+      expect(InvertedIndex.validateFileContent(validFile)).toEqual([true]);
       expect(InvertedIndex.validateFileContent(anotherValidFile)).toEqual([true]);
       expect(InvertedIndex.validateFileContent(someMalformedBooks))
-      .toEqual([false, 'malformed', [0, 2, 3, 4, 5]]);
+      .toEqual([false, 'Malformed!']);
     });
   });
 });
 
-describe('Inverted index class creates index', () => {
+describe('Inverted index class creates an index', () => {
   describe('InvertedIndex.analyse(book)', () => {
     it('returns an array of all words in book object', () => {
       let bookTokens = InvertedIndex.analyse(anotherValidFile[2]);
       expect(bookTokens).toEqual(
-        ['times', 'of', 'sciences', 'science', 'can',
-          'make', 'you', 'fly', 'with', 'an', 'edge']
+        ['science', 'can', 'make', 'you',
```

Comment:
What do you think about listing each word on seperate lines to make it easily readable and in all other similar cases?